webrender/src/picture.rs

```diff
@@ -350,13 +353,20 @@ impl PicturePrimitive {
                         let blur_render_task_id = render_tasks.add(blur_render_task);
                         self.render_task_id = Some(blur_render_task_id);
                     }
-                    Some(PictureCompositeMode::Filter(FilterOp::DropShadow(offset, blur_radius, color))) => {
+                    Some(PictureCompositeMode::Filter(FilterOp::DropShadow(_, blur_radius, color))) => {
+                        let content_width =
```

Comment:
perhaps, we can just do this one liner instead?
``` rust
let rect = (content_rect * TypedScale::<_, DevicePixel>::new(prim_context.device_pixel_ratio).round().to_i32();
```

@glennw btw, I think it would make sense to just store the `device_pixel_ratio` as that `TypedScale` in the first place